site/docs/src/json/users/search-queryString-email-database-request.json

```diff
@@ -0,0 +1,13 @@
+{
+  "search": {
+    "numberOfResults": 50,
+    "queryString": "email:@fusionauth.io",
```

Comment:
When using db search engine, prefixing with `email:` does not do anything. This query will cause the following query to be performed: `...  LIKE '%email:@fusionauth.io%`. 

To search for all users that have `@fusionauth.io` in their email address you would search using `*@fusionauth.io`. This will match a user with with `@fusionauth.io` in their email, first name, last name, middle name, or username.

Reply:
Ah, I wasn't sure if you were splitting on `:` to determine which field to look at.